test/unittests/eof_validation.cpp

```diff
@@ -14,5 +42,39 @@ void eof_validation::TearDown()
             << test_case.name << "\n"
             << hex(test_case.container);
     }
+
+    if (const auto export_dir = std::getenv("EVMONE_EXPORT_TESTS"); export_dir != nullptr)
```

Comment:
This can also be shared by having common abstract `ExportableFixture` with virtual `export_test(...)`. Can be done later though.